<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#w_login{
	border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;
}

#btn_aceptar_ingreso{
	background-color: rgb(76, 175, 80);
	border-radius: 12px;
	color: rgb(255, 255, 255);
}
#btn_aceptar_ingreso:hover{
	background-color: rgb(0, 194, 12);
}
*[objectName *=&quot;btn_manejoventana&quot;]{
	background-color: transparent;
}
*[objectName *=&quot;txt&quot;]{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="w_login" native="true">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>231</width>
        <height>161</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#w_login{
background-color: qconicalgradient(cx:1, cy:0, angle:113.6, stop:0.0397727 rgba(237, 237, 237, 255), stop:0.585227 rgba(117, 117, 117, 255));
	box-shadow: 5px 5px 50px rgba(0, 0, 0, 20);
}</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>15</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Albertus Extra Bold</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Inicio de Sesión</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_usuario">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Albertus Extra Bold</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Usuario</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_usuarioPassword">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Albertus Extra Bold</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_aceptar_ingreso">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>124</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Albertus Medium</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_panelderecho_login" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
	border-radius:25px;
}</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>26</x>
         <y>80</y>
         <width>191</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-radius: 0px;
	border-image: url(:/imgs/LOGO_KREI.png);
		box-shadow: 5px 5px 50px rgba(0, 0, 0, 20);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_manejoventana_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../LUIS_ANGEL_VALLEJO/programakw/App/Source/icons.qrc">
         <normaloff>:/icons/Icon_Close.png</normaloff>:/icons/Icon_Close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_manejoventana">
       <property name="geometry">
        <rect>
         <x>187</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../LUIS_ANGEL_VALLEJO/programakw/App/Source/icons.qrc">
         <normaloff>:/icons/Icon_Minimize.png</normaloff>:/icons/Icon_Minimize.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../LUIS_ANGEL_VALLEJO/programakw/App/Source/images.qrc"/>
  <include location="../../LUIS_ANGEL_VALLEJO/programakw/App/Source/icons.qrc"/>
 </resources>
 <connections/>
</ui>
